### PR TITLE
Add git-timemachine

### PR DESCRIPTION
**Pull request by r-darwish**

### CODE REVIEW

File: init.el [hunk 1]
```diff
@@ -788,6 +787,11 @@
   (magit-add-section-hook 'magit-status-sections-hook #'magit-insert-modules-overview nil 'append)
   (magit-define-popup-option 'magit-push-popup ?o "Set push option" "--push-option="))
 
+
```
Comment (drrlvn) [inline init.el]: Redundant newline.

File: init.el [hunk 2]
```diff
@@ -788,6 +787,11 @@
...
+(use-package git-timemachine
```
Comment (drrlvn) [inline init.el]: Didn't we use to have this? I think we did and we removed it, don't remember why.

Comment (r-darwish) [reply init.el]: You removed it in 5994742b17c2d6499e0858c823e8a0dc34a29a9b but didn't specify why 🤷‍♂

Comment (drrlvn) [reply init.el]: Maybe it slowed down emacs start up? Can you please check?

Comment (r-darwish) [reply init.el]: The init time is 1.5 with and without the package.

**Pull request merged**